configs/components/puppet-ca-bundle.json

```diff
@@ -1,4 +1,4 @@
 {
-  "url": "git://github.com/puppetlabs/puppet-ca-bundle.git",
+  "url": "git@github.com:puppetlabs/puppet-ca-bundle.git",
```

Comment:
https://github.com/puppetlabs/puppet-agent/commit/e0b4ba1b9f1db5c4caa52503d82d22a2b95d0a23 this was an intentional change made in puppet-agent#master. We should try to keep it around...